runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -1097,11 +1106,7 @@ private void saveClientBoundsConfig()
 		}
 		else
 		{
-			// Try to expand sidebar
-			if (!sidebarOpen)
-			{
-				bounds.width += pluginToolbar.getWidth();
-			}
+			final Rectangle bounds = frame.getBounds();
```

Comment:
this was merged incorrectly and now no longer compiles. I assume the removal of the above code was due to that it now remembers sidebar state so it doesn't have to offset the bounds by that anymore right?

Reply:
Correct